<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classInfoForm</class>
 <widget class="QWidget" name="classInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>管理</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_ItemClass">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>选择班级</string>
        </property>
        <item>
         <property name="text">
          <string>我的班级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>所有班级</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_Select"/>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox_ItemSelect">
        <item>
         <property name="text">
          <string>班级号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>班级名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>职工号</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_Select">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_classNum">
          <property name="text">
           <string>班级号:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_classNum"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_className">
          <property name="text">
           <string>班级名:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_className"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_classStuAmount">
          <property name="text">
           <string>人数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_classStuAmount"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_classTchAmount">
          <property name="text">
           <string>教师：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_classTchAmount"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_classSubAmount">
          <property name="text">
           <string>课程：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_classSubAmount"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>108</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_sum">
         <property name="text">
          <string>总数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_sumNum">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_count">
         <property name="text">
          <string>计数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_countNum">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_row">
         <property name="text">
          <string>行：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_rowNum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_col">
         <property name="text">
          <string>列：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_colNum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>408</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
